dotCMS/src/integration-test/java/com/dotmarketing/common/reindex/ReindexQueueAPITest.java

```diff
@@ -128,36 +128,6 @@ public void markAsFailed() throws DotDataException {
         reindexQueueAPI.markAsFailed(reindexEntry, cause);
 
         verify(reindexQueueFactory).markAsFailed(reindexEntry, cause);
-        verify(esReadOnlyMonitor).start("Reindex failed for :" + reindexEntry + " because " + cause);
-    }
-
-    /**
-     * Method to Test: {@link ReindexQueueAPIImpl#markAsFailed(ReindexEntry, String)}
-     * When: Try to mark a reindex as fail and a {@link DotDataException} in thrown
-     * Should: Should not call the {@link ESReadOnlyMonitor#start(String)} )} )} method
-     *
-     * @throws DotDataException
-     */
-    @Test
-    public void markAsFailedWhenAExceptionIsThrown() throws DotDataException {
```

Comment:
what happens now with the monitor if an exception is thrown?

Reply:
the monitor was removed from the Reindex thread in favor of a cron job